osCam/core/views.py

```diff
@@ -28,7 +28,7 @@ def get_frame(self):
             print("did not read from camera")
             time.sleep(2)
         frame = cv.flip(frame,0)
-        frame = MotionDetect.rescaleFrame(frame, .75)
+        frame = MotionDetect.rescaleFrame(frame, .25)
```

Comment:
If you would like to edit values, we should do so in a manner that allows us to move away from the whole "Magic Numbers" Anti-pattern and create a list/enum  of Constants that better define these values being used in our class. What do ya think?

Reply:
ye this was a quick fix / demonstration.
Was not what i was working on at the time, but of course magic numbers are just technical debt to pay later. 
this pr includes new variables that will be assignable in views to avoid this